docs/src/binaries/building-nym.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE HTML>
```

Comment:
remove all these `html` files from the branch please

Reply:
done